modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -100,9 +100,12 @@ const AutocompleteList = styled('ul')({
 const ResetButton = styled(IconButton)<{shouldShow: boolean}>(
   {
     width: spacing.l,
+    minWidth: spacing.l,
     height: spacing.l,
     position: 'absolute',
-    margin: 0,
+    margin: `auto ${spacing.xxxs}`,
+    top: 0,
+    bottom: 0,
```

Comment:
Previously, with `top` and `bottom` not set, things just magically centered vertically in non-IE browsers. This is an explicit way to make sure we're centering the button inside the height of the `Combobox` input.